server_tests/test/clientTests.js

```diff
@@ -1490,7 +1509,12 @@ describe("RUST API TEST SUITE", function() {
     });
 
     // BUBL CAPTURE VIDEO
-    describe("Testing /osc/commands/execute camera._bublCaptureVideo endpoint", function() {
+    describe("Testing /osc/commands/execute camera._bublCaptureVideo endpoint. Bubl exclusive test", function() {
```

Comment:
I think for the Bubl specific tests we shouldn't modify the test title at all. It's pretty self-explanatory that the "_bubl" tests will not be run for generic cameras.